### ISSUE TITLE
[docker] mysql-latest image uses postgresql

### ISSUE THREAD

**Issue by kind3r** [mikecao/umami#1191]
Docker image for mysql-v1.32.0 tries to load the posgresql provider.

```
/app/node_modules/@prisma/client/runtime/index.js:41175
--
throw new PrismaClientInitializationError(error2.message, this.config.clientVersion, error2.error_code);
^
PrismaClientInitializationError: error: Error validating datasource `db`: the URL must start with the protocol `postgresql://` or `postgres://`.
-->  schema.prisma:7
\|
6 \|   provider = "postgresql"
7 \|   url      = env("DATABASE_URL")
\|
Validation Error Count: 1
at Object.loadEngine (/app/node_modules/@prisma/client/runtime/index.js:41175:19)
at async Object.instantiateLibrary (/app/node_modules/@prisma/client/runtime/index.js:41104:5) {
clientVersion: '3.14.0',
errorCode: 'P1012'
}
```

**Comment by GiraffeCowGoat** [comment]
I'm seeing the same error. Rolling back to mysql-v1.31.0 resolves the issue.

**Comment by Lars-** [comment]
Same issue here

**Comment by mikecao** [comment]
Should be fixed now in v1.32.1
